src/lib/trpc.ts

```diff
@@ -0,0 +1,15 @@
+import { httpConnector, makeClient } from '@tianhuil/simple-trpc/dist/client'
```

Comment:
Shoudln't the client depend on the environment variable for the backend?

Reply:
We've already passed `serverUrl` and `timeout` to trpc with `guLibConfig`, I set a default value for `trpc` here to avoid build issues from needing a env variable in a shared lib.